rust/crd/Cargo.toml

```diff
@@ -1,35 +1,25 @@
 [package]
-authors = ["Lars Francke <lars.francke@stackable.de>"]
+authors = ["Stackable GmbH <info@stackable.de>"]
 description = "Contains the Trino CRD structs and utilities"
-edition = "2018"
+edition = "2021"
 license = "OSL-3.0"
 name = "stackable-trino-crd"
 repository = "https://github.com/stackabletech/trino-operator"
 version = "0.3.0-nightly"
 
 [dependencies]
-stackable-hive-crd = { git = "https://github.com/stackabletech/hive-operator.git", tag = "0.2.0" }
-stackable-opa-crd = { git = "https://github.com/stackabletech/opa-operator.git", tag = "0.5.0" }
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.4.0" }
-stackable-regorule-crd = { git = "https://github.com/stackabletech/regorule-operator.git", tag = "0.2.0" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.8.0" }
+stackable-regorule-crd = { git = "https://github.com/stackabletech/regorule-operator.git", branch = "added_cluster_ip_service" }
```

Comment:
We need the regorule crd because we create regorules (from the authorization part in the CRD). If the operator-rs differs it makes problems. This should be changed to the tag once the regorule is merged.